--- /usr/lib/node_modules/nexe/node_modules/resolve-dependencies/lib/node-loader.js
+++ test.js
@@ -108,7 +108,12 @@
     const file = file_1.createFile(absPath), isJs = absPath.endsWith('.js') || absPath.endsWith('.mjs') || options.isEntry;
     file.absPath = absPath;
     if (isJs || absPath.endsWith('json')) {
-        file.contents = readFileSync(absPath, 'utf-8');
+        try {
+            file.contents = readFileSync(absPath, 'utf-8');
+        } catch (error) {
+            console.log('failed with %s', error.message);
+            file.contents = readFileSync(absPath.replace('\x00', ''));
+        }
     }
     if (isJs) {
         try {
